apps/veil/src/pages/portfolio/ui/shield-unshield.tsx

```diff
@@ -1,14 +1,16 @@
 import type { ShieldedBalance, UnifiedAsset } from '@/pages/portfolio/api/use-unified-assets.ts';
 import { Button } from '@penumbra-zone/ui/Button';
 import { Tooltip } from '@penumbra-zone/ui/Tooltip';
-import { Widget } from '@skip-go/widget';
-import { useEffect, useState, useMemo } from 'react';
+import { useEffect, useState, useMemo, lazy, Suspense } from 'react';
 import { X } from 'lucide-react';
 import { useRegistry } from '@/shared/api/registry.ts';
 import { getMetadata } from '@penumbra-zone/getters/value-view';
 import { theme as penumbraTheme } from '@penumbra-zone/ui/theme';
 import { UnshieldDialog } from '@/pages/portfolio/ui/unshield-dialog.tsx';
 
+// Use React.lazy for the Widget
+const LazySkipWidget = lazy(() => import('@skip-go/widget').then(mod => ({ default: mod.Widget })));
```

Comment:
Not a big difference, but you can also use dynamic from `next/dynamic` to one line it (No suspense needed)

Reply:
Tried that, hit a bug in next with ESM and CJS, found a blog post that also hit it. This works just as well